admin/acceptance.py

```diff
@@ -345,25 +433,12 @@ def start_cluster(self, reactor):
             ])
             commands = commands.on(success=lambda _: zfs_commands)
 
-        certificates_path = FilePath(mkdtemp())
```

Comment:
fwiw, I wasn't super excited about this either.  We did it because it seemed like the logical extension of work started (by Tom I think) in an earlier branch and we were going to end up with a lot of duplication between the managed runner and the other runners without it.